lib/rules/web/admin_console/4.2/catalog.xyaml

```diff
@@ -1,5 +1,8 @@
 create_app_from_template:
   action: goto_create_app_from_template_page
+  action:
+    if_param: template_namespace
+    ref: goto_create_app_from_namespaced_template_page
```

Comment:
I think we can just update `goto_create_app_from_template_page` and no need to add a new action